From 72391f11f9ad941b2191eca22abe2bb5c8b8cc34 Mon Sep 17 00:00:00 2001
From: Hans-Christoph Steiner <hans@eds.org>
Date: Thu, 10 Oct 2019 12:16:28 +0200
Subject: [PATCH 3/3] Android's logging mechanism is called "logcat"

"logcat" is to Android as "syslog" is to UNIX.  It is a widely standardized
and used name throughout Android.

* https://developer.android.com/studio/command-line/logcat
* https://developer.android.com/studio/debug/am-logcat
* https://source.android.com/reference/tradefed/com/android/tradefed/util/LogcatEventParser
---
 src/app/config/config.c | 2 +-
 src/lib/log/log.c       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/app/config/config.c b/src/app/config/config.c
index 5ba187256..25954f6c9 100644
--- a/src/app/config/config.c
+++ b/src/app/config/config.c
@@ -5734,7 +5734,7 @@ options_init_logs(const or_options_t *old_options, or_options_t *options,
         goto cleanup;
       }
 
-      if (!strcasecmp(smartlist_get(elts, 0), "android")) {
+      if (!strcasecmp(smartlist_get(elts, 0), "logcat")) {
 #ifdef __ANDROID__
         if (!validate_only) {
           add_android_log(severity, options->AndroidIdentityTag);
diff --git a/src/lib/log/log.c b/src/lib/log/log.c
index 55588301f..c195c7aa8 100644
--- a/src/lib/log/log.c
+++ b/src/lib/log/log.c
@@ -1479,7 +1479,7 @@ parse_log_severity_config(const char **cfg_ptr,
         !strcasecmpstart(cfg, "stderr") ||
         !strcasecmpstart(cfg, "stdout") ||
         !strcasecmpstart(cfg, "syslog") ||
-        !strcasecmpstart(cfg, "android")) {
+        !strcasecmpstart(cfg, "logcat")) {
       goto done;
     }
     if (got_an_unqualified_range > 1)
-- 
2.20.1

